test/es6/regex-symbols.js

```diff
@@ -308,68 +385,19 @@ var tests = [
         }
     },
     {
-        name: "String.prototype.search should throw when 'this' is 'undefined'",
-        body: verifyStringSearchRequiresObjectCoercibleThis.bind(undefined),
+        name: "RegExp.prototype[@@match] exists",
+        body: verifySymbolMethodExistence.bind(undefined, Symbol.search)
```

Comment:
Looks like I forgot to change the symbol when I copied the test for `RegExp.prototype[@@search]` :smile: